src/styles/_mixins.scss

```diff
@@ -8,3 +8,16 @@
   user-select: none;
   cursor: auto;
 }
+
+@mixin font-size-26 {
```

Comment:
I think we should name the mixins with what they represent, e.g. `body-fonts` or `header-fonts` instead